### ISSUE TITLE
[Bug] Crash when navigating to a site

### ISSUE THREAD

**Issue by Amejia481** [mozilla-mobile/fenix#3025]
## Steps to reproduce
Go to www.wikipedia.org

### Expected behavior
The site is Renderer.

### Actual behavior
The app crashes.

### Stack trace
```
2019-05-30 18:45:16.939 1943-1943/org.mozilla.fenix.debug E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.fenix.debug, PID: 1943
    java.lang.ClassCastException: android.view.ViewGroup$LayoutParams cannot be cast to androidx.coordinatorlayout.widget.CoordinatorLayout$LayoutParams
        at org.mozilla.fenix.browser.BrowserFragment.onCreateView(BrowserFragment.kt:174)
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2629)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:874)
        at androidx.fragment.app.FragmentManagerImpl.addAddedFragments(FragmentManagerImpl.java:2087)
        at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManagerImpl.java:1861)
        at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManagerImpl.java:1817)
        at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManagerImpl.java:1717)
        at androidx.fragment.app.FragmentManagerImpl$2.run(FragmentManagerImpl.java:150)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6718)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

### Device information
* Android device: ?
* Fenix version: ?

**Comment by vesta0** [comment]
Can't reproduce.

**Comment by ekager** [comment]
Fixed this crash in #3024 related to swipe to refresh layout changes, closing!
